pkg/services/sqlstore/user_test.go

```diff
@@ -541,6 +541,28 @@ func TestUserDataAccess(t *testing.T) {
 					}
 				})
 			})
+
+			Convey("When creating a new user with an already existing email returns error", func() {
+				createUserCmd := &models.CreateUserCommand{
+					Email:        "user2@test.com",
```

Comment:
Where is user2@test.com already inserted into the database here? Couldn't find it.

Reply:
Sorry, on the previous version I included these tests under the `Given 5 users` group, which, in fact, is not really clear. That's the reason it wasn't clear there's a user with `Email` equals to `user", i, "@test.com` where `i` is `0...4` and the same for the `Login` attribute equals to `"loginuser", i`.

Now, I refactored the code to create an specific Convey section for these two tests.